catroid/src/main/java/org/catrobat/catroid/content/Project.java

```diff
@@ -116,6 +118,10 @@ private void ifPortraitSwitchWidthAndHeight() {
 	}
 
 	public synchronized void addSprite(Sprite sprite) {
+		if (sprite.isClone) {
+			Log.e("Project", "Do NOT add clones to the projectmanager!");
```

Comment:
Please create a log tag instead of "Project", as suggested by @robertpainsi in #1881 :

private static final String TAG = Project.class.getSimpleName();

This would be consistent with all other source files, it looks cleaner and the next time someone wants to log something, s|he can use the TAG right away 😉.